deploy/templates/nstemplatetiers/appstudio-env/cluster.yaml

```
@@ -14,9 +14,9 @@ objects:
         count/deploymentconfigs.apps: "30"
         count/pods: "150"
     selector:
-      annotations:
-        openshift.io/requester: ${SPACE_NAME}
-      labels: null
+      annotations: null
+      labels:
+        toolchain.dev.openshift.com/space: ${SPACE_NAME}
```

The format for matching the labels is different:
```suggestion
      annotations: null
      labels:
        matchLabels:
          toolchain.dev.openshift.com/space: ${SPACE_NAME}
```
That's why the e2e tests are failing - check the member operator logs:
```
ClusterResourceQuota.quota.openshift.io \"for-mig-prov-bc\" is invalid: [spec.selector: Required value: must restrict the selected projects, spec.selector.labels: Invalid value: v1.LabelSelector{MatchLabels:map[string]string(nil), MatchExpressions:[]v1.LabelSelectorRequirement(nil)}: must restrict the selected projects]
```

---

Thanks a lot for catching this @MatousJobanek 🏅 

I think I copy pasted the wrong syntax from somewhere 🤦‍♂️ , but I cannot find that reference anymore.

It's fixed  now in [95e5ea4](https://github.com/codeready-toolchain/host-operator/pull/825/commits/95e5ea4d92dbe81f3ea1e72a901b1d44938d325b). 

Let's see if tests are now passing, since I'm still having some issues setting up my temp testing env and for the same reason I was unable to test those changes the other day too.

---

e2e tests need to be updated apparently:https://gcsweb-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/origin-ci-test/pr-logs/pull/codeready-toolchain_host-operator/825/pull-ci-codeready-toolchain-host-operator-master-e2e/1671793065745977344/build-log.txt

I'm looking into it.